### PR TITLE
[Letters] Typos fix.

### PR DESCRIPTION
**Pull request by Onii-Chan-Discord**

### CODE REVIEW

File: letters/commands/letterbox.py [hunk 1]
```diff
@@ -65,9 +65,9 @@ async def delete_letter(self, ctx: commands.Context, letter_id: str):
             return
         msg = await ctx.send(
             bold(
-                "Are you sure you want to delete this letter? This action "
-                + underline("cannot be")
-                + " recovered.",
+                "Are you sure you want to delete this letter? You "
+                + underline("won't be")
+                + "  able to recovere this letter afterwards.",
```
Comment (Predeactor) [inline letters/commands/letterbox.py:70]: There is a double space here.

File: letters/commands/letterbox.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ async def delete_letter(self, ctx: commands.Context, letter_id: str):
             await self.config.custom("LetterBox", ctx.author.id).letterbox.clear_raw(letter_id)
             await msg.edit(content="Deleted your letter.")
         else:
-            await msg.edit(content="I won't touch it then!")
+            await msg.edit(content="Cancelling deletion")
```
Comment (Predeactor) [inline letters/commands/letterbox.py:84]: I don't like this edit, please revert it.

File: letters/commands/writer.py [hunk 3]
```diff
@@ -76,16 +76,16 @@ async def confirm(self, ctx: commands.Context, content: str):
     async def ask_for_receivers(self, ctx: commands.Context) -> Optional[dict]:
         # ask if know to who to send to
         result = await self.confirm(
-            ctx, "Do you know the IDs/names of the persons I must contact?"
+            ctx, "Do you know the IDs/names of the people I must contact?"
```
Comment (Predeactor) [inline letters/commands/writer.py:79]: "people" is not a term I agree with, this make the sentence strange to read.

File: letters/commands/writer.py [hunk 4]
```diff
@@ -76,16 +76,16 @@ async def confirm(self, ctx: commands.Context, content: str):
...
         )
         if result is False:
             raise NeedCHPostal()
         # ask for id
         message = (
-            "Please give me the name, mention, or ID of the persons you want to send a letter "
-            "to.\n\nThis need to be separated with a command, for example:\n"
+            "Please give me the name, mention, or ID of the people you want to send a letter "
```
Comment (Predeactor) [inline letters/commands/writer.py:85]: This is even more weird.

File: letters/commands/writer.py [hunk 5]
```diff
@@ -156,11 +156,11 @@ async def go_for_letter(self, ctx: commands.Context):
             await ctx.send("Feel free to ask me another day!")
             return
         except ArgumentParsingError:
-            await ctx.send("I can't send a letter to more than 5 persons, sorry!")
+            await ctx.send("I can't send a letter to more than 5 people, sorry!")
```
Comment (Predeactor) [inline letters/commands/writer.py:159]: Same as before.

File: letters/commands/writer.py [hunk 6]
```diff
@@ -108,14 +108,14 @@ async def ask_for_receivers(self, ctx: commands.Context) -> Optional[dict]:
             return None
 
         # check if valid
-        msg = "I have found the following users, is that right?\n" + "\n".join(
+        msg = "I have found the following users, are they the correct users?\n" + "\n".join(
```
Comment (Predeactor) [inline letters/commands/writer.py:111]: Is there a space between "user" and "s"? If so please fix that.

**Review by Predeactor — CHANGES_REQUESTED** [review]
The following comments are what I do not appreciate in this PR.
I would like the revert of the "people" change most especially, it doesn't fit well at all.

File: letters/commands/writer.py (hunk 6 above)
Comment (Predeactor) [reply letters/commands/writer.py:111]: Was just my phone showing me weird things...

File: letters/commands/writer.py (hunk 4 above)
Comment (Onii-Chan-Discord) [reply letters/commands/writer.py]: So should i revert this change?

**Comment by Onii-Chan-Discord** [comment]
Fixed all of the requested changes except 1.

File: letters/commands/writer.py (hunk 4 above)
Comment (Predeactor) [reply letters/commands/writer.py]: Yes please do.

**Comment by Onii-Chan-Discord** [comment]
I believe all requested changes have been made

**Review by Predeactor — APPROVED** [review]

**Comment by Predeactor** [comment]
Thank for the PR.

**Pull request merged**